Kinvey/Kinvey/CustomEndpoint.swift

```diff
@@ -103,11 +134,18 @@ open class CustomEndpoint {
     
     /// Executes a custom endpoint by name and passing the expected parameters.
     @discardableResult
-    open static func execute(_ name: String, params: Params? = nil, client: Client = sharedClient, completionHandler: CompletionHandler<JsonDictionary>? = nil) -> Request {
+    open static func execute(
+        _ name: String,
+        params: Params? = nil,
+        client: Client = sharedClient,
+        requestConfiguration: RequestConfiguration? = nil,
+        completionHandler: CompletionHandler<JsonDictionary>? = nil
+    ) -> Request {
         return execute(
             name,
             params: params,
-            client: client
+            client: client,
```

Comment:
I think it's a good idea, but it will result in a huge breaking change. Do you think it's worth doing it? I think it's worth it because I'm don't seeing many people using a custom reference for the `client` anyway.